prow/plugins/trigger/ic.go

```diff
@@ -65,12 +65,16 @@ func handleIC(c client, ic github.IssueCommentEvent) error {
 		return err
 	}
 
+	trustedOrg := org
```

Comment:
I didn't add a default trusted org. Instead, if there is no trusted org set for a repo then no users are trusted.